/**:
  ros__parameters:
    model_name: TinyLidarNet ## TinyLidarNet or TinyLidarLstmNet
    model_path: .pt
    max_range: 30.0
    input_dim: 1081
    output_dim: 2